_yard/API.erb

```diff
@@ -161,15 +169,27 @@ curl -s -F password="admin" "http://localhost:8089/users/admin/login"
 }
 ```
 
-> It's a good idea to save the "session" id, since this will be used for later requests.
+> It's a good idea to save the session key since this will be used for later requests:
+
+```shell
+# Mac/Unix terminal
+export SESSION="9528190655b979f00817a5d38f9daf07d1686fed99a1d53dd2c9ff2d852a0c6e"
+
+# Windows Command Prompt
+set SESSION="9528190655b979f00817a5d38f9daf07d1686fed99a1d53dd2c9ff2d852a0c6e"
+
+# Windows PowerShell
+$env:SESSION="9528190655b979f00817a5d38f9daf07d1686fed99a1d53dd2c9ff2d852a0c6e"
+```
+
+```python
```

Comment:
I think it might make sense to do one of more of the following here:

1. link to the section of the ASnake doc dealing with auth: https://github.com/archivesspace-labs/ArchivesSnake/#low-level-api

2. Show handling it in raw requests (i.e. creating a session and setting the appropriate header)

Reply:
Sooo.. I think option 1 is the correct solution @pobocks , but I can't for the life of me figure out how to get a clickable link rendered inside that inline code block that it's gotta be in for the tabbed rendering to work.  Thoughts?

I'm gonna update this branch to make all the docs now-docs like we talked about a bit ago, but if you have insight here on getting that link to work lemme know!

Reply:
I don't think there's a way to get it _inside_ a code block - it would need to go somewhere Not Inside Code Blocks if you wanted it clickable.  Not 100% sure if "clickable but people see see a link that's not relevant to them if they're using something else, or 

```python
# more information on auth handling here: https://github.com/archivesspace-labs/ArchivesSnake/#low-level-api

def code(arg):
    code(code(code)) 
```